### ISSUE TITLE
as a repository operator, I'd like search engine discoverability to not break

### ISSUE THREAD

**Issue by pameyer** [IQSS/dataverse#5406]
Spin-off from https://github.com/IQSS/dataverse/issues/5400 ; having multiple `<head>` elements in generated HTML may prevent search engines from correctly parsing meta-tags, schema.org json-ld and other machine-readable parts of pages that used to assist discoverability.

It would seem reasonable that any pages a dataverse installation would like to have show up in search results to be checked.

Pages:
- dataset page (should be ok once https://github.com/IQSS/dataverse/pull/5405 is merged)
- dataverse page
- (custom?) home page
- other pages?

How to check:
- no fatal errors with vnu (https://validator.github.io/validator/), or online variant.  Getting pages to 0 errors is explicitly out of scope here; goal is to have the generated html parsable to the point of being able to report how many errors there are.
- visual inspection?
- other / better ways?

**Comment by mheppler** [comment]
Pretty sure this is just another symptom of the same problem reported in #5400.

Vote to close.

**Comment by pameyer** [comment]
@mheppler same problem; but if it's a dupe or not depends on where the scope boundaries for #5400 get drawn.

**Comment by pameyer** [comment]
With the approach taken in #5407 cbc7dd62a90a4fb796e942f26e8037aee6b5cf74 , this is a functional duplicate of #5400
